lib/pacemaker/pcmk_verify.c

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright 2023 the Pacemaker project contributors
+ *
+ * The version control history for this file may have further details.
+ *
+ * This source code is licensed under the GNU Lesser General Public License
+ * version 2.1 or later (LGPLv2.1+) WITHOUT ANY WARRANTY.
+ */
+
+#include <crm_internal.h>
+#include <crm/cib/internal.h>
+#include <crm/common/output.h>
+#include <crm/common/results.h>
+#include <crm/common/scheduler.h>
+#include <pacemaker-internal.h>
+#include <pacemaker.h>
+
+#include <stdint.h>
+#include <sys/types.h>
+#include <sys/stat.h>
+#include <unistd.h>
+
+#include "libpacemaker_private.h"
+
+int
+pcmk__parse_cib(pcmk__output_t *out, const char *cib_source, xmlNodePtr *cib_object)
+{
+    int rc = pcmk_rc_ok;
+
+    crm_info("=#=#=#=#= Getting XML =#=#=#=#=");
```

Comment:
I assume this was just for debugging, it can go

Reply:
It was in the original code, so removing it would change the output. But I can still remove it if you want (we are changing the output anyway).

Reply:
Yes, I think so. It's just a log message, and it doesn't add any useful info.